src/main/models/terminal/TerminalWorker.java

```diff
@@ -0,0 +1,41 @@
+package main.models.terminal;
+
+import main.models.terminal.interfaces.Display;
+import main.models.terminal.interfaces.Park;
+import main.models.vehicles.HybridCar;
+import main.models.vehicles.Vehicle;
+
+public class TerminalWorker implements Park, Display {
```

Comment:
Be careful: 'basic' worker earns `10%` while 'advanced' worker earns `11%` of the profit of each card. Because of that, it would be more appropriate to subclass into `BasicWorker` and `AdvancedWorker`.

Reply:
Worker is now subclassed